scripts/e2e/e2e.sh

```diff
@@ -184,6 +188,11 @@ go get -u github.com/vmware/govmomi/govc
 # Push new container images
 # TODO the `-k` flag here is a workaround until we can set GCR_KEY_FILE properly
 hack/release.sh -t pr -p -k /root/.capv/keyfile.json
+
+# export variables needed to generate spec files
+vsphere_controller_version="gcr.io/cluster-api-provider-vsphere/pr/manager:${VERSION}"
```

Comment:
Since this line is now identical to Line 126 (we set it as a `local` in `apply_secret_to_bootstrap`, let's just make a full-on global var, setting `VSPHERE_CONTROLLER_VERSION` right after we set `VERSION`, and then also export it. That we con't have to set it a second time in `apply_secret_to_bootrap` -- that `local` far can go away.

Though I do see we are exporting that env var twice -- once in plain text, and once in `base64`... clearly things are working, but that seems like a problem waiting to happen. Might be worth seeing if there's something we can do about that... Maybe export the `base64` encoded version with a different name, and edit the secret template to use that instead?

Anyways, this is also a suggestion rather than a requirement. A future cleanup that can be made. :)